src/main/java/org/takes/rs/RsWithType.java

```diff
@@ -40,6 +40,11 @@
 public final class RsWithType extends RsWrap {
 
     /**
+     * Type header.
+     */
+    private static final String HEADER = "Content-Type";
```

Comment:
Hi @carlosmiranda :)
(Thanks for th review)
This constant is actually used twice, and I had to make it a constant to pass the static analysis. So, IMHO, this is ok.